lib/oath/views/account_view.dart

```diff
@@ -99,6 +99,24 @@ class _AccountViewState extends ConsumerState<AccountView> {
     }).toList();
   }
 
+  String? a11yCredentialLabel(String? issuer, String? name, OathCode? code) {
```

Comment:
This should probably be a private function (rename to `_ally...`) and doesn't need to be part of the class, I'd move it out.